src/Responses/PsrResponseFactory.php

```diff
@@ -54,14 +53,6 @@ public function create(FilesystemOperator $cache, $path)
         $cacheControl = 'max-age=31536000, public';
         $expires = date_create('+1 years')->format('D, d M Y H:i:s').' GMT';
 
-        if (false === $contentType) {
-            throw new FilesystemException('Unable to determine the image content type.');
-        }
-
-        if (false === $contentLength) {
-            throw new FilesystemException('Unable to determine the image content length.');
-        }
```

Comment:
I hope you don't mind me asking but doesn't removing this cause any issues? Because file size van still be `null`?

https://github.com/thephpleague/flysystem/blob/7cbbb7222e8d8a34e71273d243dfcf383ed6779f/src/FileAttributes.php#L72-L75

Someone might be catching this.

Reply:
`FileAttributes` is an internal class. `Filesystem::fileSize()` has it's return value typed as `int` https://github.com/thephpleague/flysystem/blob/7cbbb7222e8d8a34e71273d243dfcf383ed6779f/src/Filesystem.php#L118. So it won't return `null`.

Reply:
There's a possibility that the adapter could throw an exception if it's unable to access the file, so we could wrap those calls in try/catch and throw own own `League\Glide\Filesystem\FilesystemException` exception if we want to.

Reply:
I don't think try catching here is necessary: this probably changed in Flysystem 2.0 anyway, if people upgrade to Glide 2.0 they'll need to adapt their code here, that's fine IMO.